From e7529877a4d4c76748b6b2dc4a92631463bc14e5 Mon Sep 17 00:00:00 2001
From: Eric Ruei <e-ruei1@ti.com>
Date: Tue, 27 Mar 2018 17:36:20 -0400
Subject: [PATCH 206/210] arm: dts: am571x-idk-pps: add gmac/cpts pps
 output-controlled gpios

A gpio-controlled multiplexer is added to guarantee that there is only
one PPS output at any time at the AM571(2)x-idk. There is also a
gpio-controlled AND gate added for CPTS 1PPS reference signal control.

This patch adds GMAC/CPTS-related gpio entries.

Signed-off-by: Eric Ruei <e-ruei1@ti.com>
---
 arch/arm/boot/dts/am571x-idk-pps.dts | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/am571x-idk-pps.dts b/arch/arm/boot/dts/am571x-idk-pps.dts
index 96ca092..b367bfd 100644
--- a/arch/arm/boot/dts/am571x-idk-pps.dts
+++ b/arch/arm/boot/dts/am571x-idk-pps.dts
@@ -19,6 +19,9 @@
 	pinctrl-3 = <&pps1_timer16_pwm2_on>;
 	pinctrl-4 = <&pps1_timer15_latch_on>;
 	pinctrl-5 = <&pps1_timer15_latch_off>;
+
+	pps-enable-gpios = <&gpio1 5 GPIO_ACTIVE_HIGH>;
+	ref-enable-gpios = <&gpio6 19 GPIO_ACTIVE_HIGH>;
 };
 
 &pruss1_eth {
-- 
1.9.1

